src/components/onboarding/screens/username.tsx

```diff
@@ -44,15 +46,21 @@ interface UsernameProps {
 }
 
 export const Username: React.FC<UsernameProps> = ({ next }) => {
-  const title = 'Choose a username';
   const dispatch = useDispatch();
   const { name } = useAppDetails();
 
-  const { wallet, screen } = useSelector((state: AppState) => ({
+  const { wallet, screen, secretKey, appName } = useSelector((state: AppState) => ({
     wallet: selectCurrentWallet(state) as Wallet,
     screen: selectCurrentScreen(state),
+    secretKey: selectSecretKey(state),
+    appName: selectAppName(state),
   }));
 
+  const title = `${appName} with Blockstack`;
```

Comment:
I definitely appreciate the intention here, but changing the actual screen title makes this page much more confusing. It went from "Choose a Username" to "AppName with Blockstack". Did you only mean to update the `document.title`? There must be a way to use a hidden label, or something, to do this without updating the page's copy.

![image](https://user-images.githubusercontent.com/1109058/75200061-9b541800-5719-11ea-8c30-1ba1dfea2ed8.png)

Reply:
Open to suggestions, I'd love another way of doing it. This less-than-ideal title hack is better than having a password entry saved as "Choose an account", no? [Asked @1password on Twitter](https://twitter.com/kyranjamie/status/1231936609240649728?s=20)

Reply:
I also tried changing on submit, but it has to happen before then